src/guide/advanced/intro.md

```diff
@@ -96,17 +96,16 @@ edition = "2021"
 crate-type = ['cdylib']
 
 [dependencies]
-iroha_data_model = { git = "https://github.com/hyperledger/iroha/", branch = "iroha2", default-features = false }
 iroha_wasm = { git = "https://github.com/hyperledger/iroha/", branch = "iroha2" }
 
 parity-scale-codec = {version = "2.3.1", default-features = false }
 ```
 
-Note that the crate type is `cdylib`. WASM is a portable format, most Rust code is linked in a non-portable architecture- and OS-specific static manner. As Rust does not (at present) have a stable ABI, we have to rely on the C-linkage to generate WASM. You don't have to worry about C foreign function interfaces too much in the smart contract itself, we took care of most of that for you.
+Note that the crate type is `cdylib`. Most Rust code is linked in a non-portable architecture and OS-specific static manner, but WASM is a portable format. Since C ABI is the lingua franca of the programming world and there is no other stable Rust ABI, Iroha relies on the C-linkage to generate WASM bindings. Note that you don't have to worry about C FFI in the smart contract itself, most of that is taken care for you.
```

Comment:
```suggestion
Note that the crate type is `cdylib`. Most Rust code is linked in a non-portable architecture and OS-specific static manner, but WASM is a portable format. Since C ABI is the *lingua franca* of the programming world and there is no other stable Rust ABI (yet), Iroha relies on the C-linkage to generate WASM bindings. Thankfully, you don't need to worry about foreign function interface (FFI)-related problems like `unsafe`, `repr(C)`, padding, alignment etc., `iroha_wasm` takes care of all that for you. 
```